Clean up post-work TODOs and Content Sweep

leerogers-bjss opened the pull request:
Content changes requested by luke from post-work content sweep.

leerogers-bjss committed:
Clean up post-work TODOs

Content changes requested by luke from post-work content sweep.

bjss-colin commented on src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Order/Views/Quantity/SelectServiceRecipientQuantity.cshtml:
```diff
@@ -31,10 +35,10 @@
 			<input type="hidden" asp-for="@Model.BillingPeriod" />
 
-            <h3>Quantity per practice</h3>
+            <h2>@x</h2>
```
Needs a better variable name, also, should this be assigned to `label-text` on the `nhs-table` tag as we do with all other table headings?

leerogers-bjss replied:
doh, this was from when i was showing luke an example.

i'll get it fixed now

leerogers-bjss committed:
small fix

change variable name from x to be more descriptive on service recipient quantity page

change variable to take in range description for h2 title on service recipient quantity page

fix tests for FundingSource model

bjss-colin commented on src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Order/Models/SolutionSelection/Quantity/SelectServiceRecipientQuantityModel.cs:
```diff
@@ -58,7 +58,7 @@ public SelectServiceRecipientQuantityModel(OrderItem orderItem)
         {
             ProvisioningType.Patient => "Practice list size",
-            _ => "Quantity",
+            _ => RangeDefinition,
```
This should be "Quantity". I agreed this with Luke yesterday as it doesn't read well if the range definition has been entered as lowercase.

leerogers-bjss committed:
Fix tests and update how the quantity column is set
